frontend/resourceadm/pages/MigrationPage/MigrationPage.tsx

```diff
@@ -85,13 +91,15 @@ export const MigrationPage = () => {
           </div>
         </div>
         <div className={classes.migrateBox}>
-          <h3 className={classes.header3}>Velg tidspunkt for migrering</h3>
-          <p className={classes.text}>
+          <Heading size='xsmall' spacing level={3}>
+            Migrering av tjeneste
+          </Heading>
+          <Paragraph size='small'>
             Jeg forstår at å trykke <strong>start migrering</strong> fører til at følgende
             handlinger iverksettes.
-          </p>
+          </Paragraph>
           <div className={classes.migrateSteps}>
-            <p>TODO steps</p>
+            <Paragraph size='small'>TODO steps</Paragraph>
```

Comment:
Is this text display intentional?

Reply:
Yes, the page is not visible to anyone yet. It will come on next pr